logging/src/main/resources/config/liquibase/changelog/logback/mysql.sql

```diff
@@ -0,0 +1,53 @@
+# Logback: the reliable, generic, fast and flexible logging framework.
```

Comment:
How does the license work in this case? Can we just include licensed code in our repo?

Reply:
I've checked it and its allowed. 

It's not possible to reference the existing files in their library, due to small but necessary changed made to these files.